sdk-docs/save-data-from-your-app/using-a-custom-field.md

```diff
@@ -16,7 +16,7 @@ npm install @trackunit/iris-app-runtime-core
 
 
 
-2. Use the `CustomFieldRuntime` command to retrieve custom fields definitions and values saved to the specified Asset ID. If there are no saved values of the Asset ID, only the definition will be retrieved.
+2. Use the `CustomFieldRuntime` command to retrieve custom fields definitions and values saved to the specified `Asset ID`. If there are no saved values of the Asset ID, only the definition will be retrieved.
```

Comment:
Here you changed Asset ID to `Asset ID` in one instance but not the other. I wonder if we are consistent in the way we use `code blocks`? Shouldn't it only be used to reference code?

Reply:
For readability, I like it when people use `` to highlight variables, function names, commands, etc. and other key terms. But I keep it to only the first time it is introduced/mentioned, as it becomes cluttered otherwise. This is a personal preference, but I'll continue styling these docs the way I like it. Once I've written (and rewritten) a good amount of docs, I'll take some feedback from everyone on what works and doesn't work and I'll try to condense it into specific conventions that we'll all adhere to then :)